packages/docsearch-react/src/AskAiScreen.tsx

```diff
@@ -120,127 +126,113 @@ function AskAiExchangeCard({
-                              className="DocSearch-AskAiScreen-MessageContent-Tool Tool--PartialCall shimmer"
-                            >
-                              <LoadingIcon className="DocSearch-AskAiScreen-SmallerLoadingIcon" />
-                              <span>{translations.preToolCallText || 'Searching...'}</span>
-                            </div>
-                          );
-                        case 'call':
-                          return (
-                            <div key={index} className="DocSearch-AskAiScreen-MessageContent-Tool Tool--Call shimmer">
-                              <LoadingIcon className="DocSearch-AskAiScreen-SmallerLoadingIcon" />
-                              <span>
-                                {`${translations.duringToolCallText || 'Searching for '} "${toolInvocation.args?.query || ''}" ...`}
-                              </span>
-                            </div>
-                          );
-                        case 'result':
-                          return (
-                            <div key={index} className="DocSearch-AskAiScreen-MessageContent-Tool Tool--Result">
-                              <SearchIcon size={18} />
-                              <span>
-                                {`${translations.afterToolCallText || 'Searched for'}`}{' '}
-                                <span
-                                  role="button"
-                                  tabIndex={0}
-                                  className="DocSearch-AskAiScreen-MessageContent-Tool-Query"
-                                  onKeyDown={(e) => {
-                                    if (e.key === 'Enter' || e.key === ' ') {
-                                      e.preventDefault();
-                                      onSearchQueryClick(toolInvocation.args?.query || '');
-                                    }
-                                  }}
-                                  onClick={() => onSearchQueryClick(toolInvocation.args?.query || '')}
-                                >
-                                  {' '}
-                                  &quot;{toolInvocation.args?.query || ''}&quot;
-                                </span>
-                              </span>
-                            </div>
-                          );
-                        default:
-                          return null;
-                      }
-                    }
-                    // fallback for unknown tool, should never happen in theory. :shrug:
+                  case 'output-available':
                     return (
-                      <span key={index} className="text-sm italic shimmer">
-                        {translations.thinkingText || 'Thinking...'}
-                      </span>
+                      <div key={index} className="DocSearch-AskAiScreen-MessageContent-Tool Tool--Result">
+                        <SearchIcon />
```

Comment:
thanks for removing aggregated tool calling, it is not so useful and sometimes doesn't work